### DISCUSSION TITLE
Option for Toggle on push and Slider on slide

### DISCUSSION THREAD

**Discussion by snowfox-design** [mattieha/slider-button-card#235] [Ideas]
First off, thank you for a great card. I use this daily with my lights. 

I would like to suggest a feature allow a user to tap and release anywhere within the card, and it acts as a toggle (like it does now) but if you tap and swipe before releasing, it operates the slide functon. Allowing for two functions on the same button.

I know you can have a toggle button configured but we use these as panel cards on 5" touchscreens in place of light switches so we want to keep the toggle button area as large as possible (ideally the whole screen). 

Please tell me if this already exists and i missed it in the documentation. Would be great to see if anyone else would find this useful.

**Comment by Nickxsch** [comment]
I've created a work-around by duplicating the card:
1: works as a toggle, is only visible when entity is off.
2: works as a slider, is only visible when entity is on.

Works alright, but not intuitive, as tapping it to turn it off does not work this way. So yes, this would be a great feature!
